code/modules/clothing/head/mind_monkey_helmet.dm

```diff
@@ -0,0 +1,107 @@
+/obj/item/clothing/head/mind_monkey_helmet/monkey_sentience
+	name = "monkey mind magnification helmet"
+	desc = "A fragile, circuitry embedded helmet for boosting the intelligence of a monkey to a higher level. You see several warning labels..."
+
+	base_icon_state = "monkeymind"
+	icon_state = "monkeymind1" //For mapping?
+	item_state = "monkeymind"
+	strip_delay = 100
+	var/cooldown_expiry //It'll get annoying quick when someone tries to remove their own helmet 20 times a second
+	var/mob/living/carbon/monkey/magnification = null ///if the helmet is on a valid target (just works like a normal helmet if not (cargo please stop))
+	var/polling = FALSE///if the helmet is currently polling for targets (special code for removal)
+
+/obj/item/clothing/head/mind_monkey_helmet/monkey_sentience/Initialize()
+	. = ..()
+	base_icon_state = "[base_icon_state][rand(1,3)]"
+	update_icon()
+
+/obj/item/clothing/head/mind_monkey_helmet/monkey_sentience/update_icon(updates)
+	. = ..()
+	if (magnification)
+		icon_state = "[base_icon_state]up"
+		return
+	icon_state = "[base_icon_state]"
+
+
+/obj/item/clothing/head/mind_monkey_helmet/monkey_sentience/examine(mob/user)
+	. = ..()
+	. += "<span class='boldwarning'>---WARNING: REMOVAL OF HELMET ON SUBJECT MAY LEAD TO:---</span>"
+	. += "<span class='warning'>BLOOD RAGE</span>"
+	. += "<span class='warning'>BRAIN DEATH</span>"
+	. += "<span class='warning'>PRIMAL GENE ACTIVATION</span>"
+	. += "<span class='warning'>GENETIC MAKEUP MASS SUSCEPTIBILITY</span>"
+	. += "<span class='boldnotice'>Ask your CMO if mind magnification is right for you.</span>"
+
+/obj/item/clothing/head/mind_monkey_helmet/monkey_sentience/equipped(mob/user, slot)
+	. = ..()
+	if(slot != ITEM_SLOT_HEAD)
+		return
+	if(!ismonkey(user) || user.ckey)
+		var/mob/living/something = user
+		to_chat(something, "<span class='boldnotice'>You feel a stabbing pain in the back of your head for a moment.</span>")
+		something.apply_damage(5,BRUTE,BODY_ZONE_HEAD,FALSE,FALSE,FALSE) //notably: no damage resist (it's in your helmet), no damage spread (it's in your helmet)
+		playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
+		return
```

Comment:
after this part, the codes should be in a separated proc, and be called by `INVOKE_ASYNC`
Doing this without async means `equipped()` is damn delayed but that shouldn't be a thing.